src/ui/component/walletSend/view.jsx

```diff
@@ -35,6 +35,10 @@ class WalletSend extends React.PureComponent<Props> {
   render() {
     const { balance } = this.props;
 
+    if (!balance) {
```

Comment:
I disagree with this. Let me learn that it is possible and what it looks like. Put the form and/or buttons in a non-interactive state if we think it's poor UX to display a form that cannot be submitted.